lib/output.js

```diff
@@ -511,6 +516,80 @@ function gif (options) {
   return this._updateFormatOut('gif', options);
 }
 
+/**
+ * Use these JP2 options for output image.
+ *
```

Comment:
Please can you add info about the lack of prebuilt support for this, something like:
```
 * Requires libvips compiled with support for OpenJPEG.
 * The prebuilt binaries do not include this - see
 * {@link https://sharp.pixelplumbing.com/install#custom-libvips installing a custom libvips}.
```